### PR TITLE
Updated and cleaned up the Turbonomic Labs

### PR DESCRIPTION
**Pull request by r2raman**
Fixes | Relates to **[issue link]**

Content changes:

- Any content updates here

Code changes:

- Any code changes here

### CODE REVIEW

File: content/Turbonomic/README.md [hunk 1]
```diff
@@ -11,13 +11,11 @@ After completing these labs in a workshop session, you'll be able to earn the [T
...
+- Complete [Turbonomic by IBM Sales Foundation](https://learn.ibm.com/course/view.php?id=9929)
+- [A TechZone Reservation](https://learn.ibm.com/mod/page/view.php?id=210016). Follow the instruction in this document to reserve the Turbonomic Instance.
 
-      <br/> **Reservation Type**: _Reserve Now_
-      <br/> **Name**: _Turbonomic for Workshop_
-      <br/> **Purpose**: _Practice/Self-Education_
-      <br/> **Preferred Geography**: _AMERICAS - us-south-region - dal10_
+> Note: Turbonomic instances will be reserved by Instructor for VEST Workshop.
 
 ## Labs
```
Comment (maurelya) [inline content/Turbonomic/README.md:20]: Can you add Module lab 106  to the list of Labs?

File: content/Turbonomic/101.md [hunk 2]
```diff
@@ -1,37 +1,42 @@
 ---
-title: '101: Turbonomic Overview'
+title: '101: Turbonomic Overview and Explore UI'
 timeToComplete: 20
 updated: 2023-05-19
```
Comment (maurelya) [inline content/Turbonomic/101.md:4]: Updated date should be current

File: content/Turbonomic/102.md [hunk 3]
```diff
@@ -1,87 +1,56 @@
 ---
-title: '102: Explore the Turbonomic UI'
+title: '102: Optimizing Cost'
 timeToComplete: 15
 updated: 2023-05-01
```
Comment (maurelya) [inline content/Turbonomic/102.md:4]: Updated date should be current

File: content/Turbonomic/103.md [hunk 4]
```diff
@@ -1,87 +1,120 @@
 ---
-title: '103: Connect Instana to Turbonomic'
-timeToComplete: 25
+title: '103: Business Application Drill Down'
+timeToComplete: 40
 updated: 2023-05-01
```
Comment (maurelya) [inline content/Turbonomic/103.md:4]: Updated date should be current

File: content/Turbonomic/104.md [hunk 5]
```diff
@@ -1,188 +1,81 @@
 ---
-title: '104: Turbonomic Actions and Policies'
-timeToComplete: 30
+title: '104: Infrastructure Changes Planning'
+timeToComplete: 40
 updated: 2023-05-01
```
Comment (maurelya) [inline content/Turbonomic/104.md:4]: Updated date should be current

File: content/Turbonomic/105.md [hunk 6]
```diff
@@ -1,150 +1,87 @@
 ---
-title: '105: Planning and Custom Dashboards'
-timeToComplete: 20
+title: '105: Connect Instana to Turbonomic'
+timeToComplete: 25
 updated: 2023-05-01
```
Comment (maurelya) [inline content/Turbonomic/105.md:4]: Updated date should be current

File: content/Turbonomic/106.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
+---
+title: '106: Turbonomic Actions and Policies'
+timeToComplete: 30
+updated: 2023-05-01
```
Comment (maurelya) [inline content/Turbonomic/106.md:4]: Updated date should be current

File: content/Turbonomic/README.md (hunk 1 above)
Comment (maurelya) [reply content/Turbonomic/README.md:20]: The Module names should reflect the same module names in the labs

**Review by maurelya — CHANGES_REQUESTED** [review]

File: content/Turbonomic/README.md (hunk 1 above)
Comment (r2raman) [reply content/Turbonomic/README.md:20]: Done

File: content/Turbonomic/101.md (hunk 2 above)
Comment (r2raman) [reply content/Turbonomic/101.md:4]: changed

File: content/Turbonomic/103.md (hunk 4 above)
Comment (r2raman) [reply content/Turbonomic/103.md:4]: changed

File: content/Turbonomic/104.md (hunk 5 above)
Comment (r2raman) [reply content/Turbonomic/104.md:4]: changed

File: content/Turbonomic/105.md (hunk 6 above)
Comment (r2raman) [reply content/Turbonomic/105.md:4]: changed

File: content/Turbonomic/106.md (hunk 7 above)
Comment (r2raman) [reply content/Turbonomic/106.md:4]: changed

**Pull request merged**